day2.md

```diff
@@ -0,0 +1,171 @@
+<br/><img src="day2/IMG_20160906_121403.jpg" width="500"><br/>
+First thing we saw after we got out from the station was a store for this famous banana bread.
+
+The bottom floors of the skytree tower includes various areas such as a shopping district, a fashion zone, and a food court.
+
+We randomly walked into a manga/anime shop packed with manga and related products! <br />
+Grown up watch anime, the place recalled a lot of good old time for me.
+
+<img src="day2/IMG_20160906_124543.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_124559.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_124714.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_124721.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_124747.jpg" width="500"><br/>
+So many One Piece stuff!!! 😱😱😱<br/>
+
+<img src="day2/IMG_20160906_125154.jpg" width="500"><br/>
+<img src="day2/14435302_1115753031807723_6380156596113072249_o.jpg" width="500"><br/>
+They labeled the actual heights of some famous manga characters.<br />
+I'm pretty sure 越前リョーマ from The Prince of Tennis, the anime that got me into playing tennis, grown over 151cm after middle school.😓<br />
+Matt is about the same heigh as Luffy! lol 😂
+
+<img src="day2/IMG_20160906_132543.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_132619.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_132948.jpg" width="500"><br/>
+One of my favorite anime: Ojarumaru (反斗小王子)! I watched it when I was in preschool living with my aunt and cousin! 😄 I even watched a bit of it on my phone on Youtube when we got back to our airbnb place that day just to recall some good old memories. 😂
+
+<img src="day2/14380199_1115753295141030_5126574712730509424_o.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_133019.jpg" width="500"><br/>
+Don't know what's his name, but I saw him everwhere and he is so cute!
+
+Then we went to get in the line for getting tickets to go up to the skytree tower. We did some research and watched a Youtube video about the view on top of the tower while waiting in line, and we decided to pass it. 😂
+
+Before we left, we decided to check out the food court in the bottom of the building and we so glad we did!
+<img src="day2/IMG_20160906_133204.jpg" width="500"><br/>
+
+The food court is packed with **a lot of** food shops: including sushi, all kinds of rice and cooked dishes, and sweets, all in incredible prices! Hear the 5-years-old inside me screaming?!?! 😱😱😱
+
+<img src="day2/IMG_20160906_133318.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_133344.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_133356.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_141643.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_134513.jpg" width="500"><br/>
+<img src="day2/IMG_20160906_141810.jpg" width="500"><br/>
+
+
+
+<img src="day2/IMG_20160906_135506.jpg" width="500"><br/>
+These colorful rice intrigued me a lot. So I got them along with tofu and pork bellies!
```

Comment:
"**This** colorful rice"

In #11 of [this article](http://theweek.com/articles/459959/11-nouns-that-only-have-plural-form):
> For example, rice, sand, sugar, and salt are all mass nouns. Mass nouns have singular verb agreement ("the rice is cooked").